### PR TITLE
Migrate ipFamily fields to k8s >= 1.20

### LINKED ISSUE #4729: k8s 1.20 fails on IPV6 tests
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:

A test run of the new 1.20 lane failed with some test failures for IPV6:

https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4662/pull-kubevirt-e2e-k8s-1.20/1344200275589599232

/cc @phoracek @qinqon 

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
At the k8s 1.20 version there is a backward incompatible change realted
to ipFamily field at service [1], this change detects if kubevirt is
running at >= 1.20 and translate ipFamily field to ipFamilies.

[1] https://kubernetes.io/docs/setup/release/notes/#api-change-3

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #4729 

**Special notes for your reviewer**:
The alternative to this is to bump the API so it has the `ipFamilies` field and do the opposite migration at https://github.com/kubevirt/kubevirt/pull/4870.

**Release note**:

```release-note
Migrate ipFamily fields to ipFamilies at k8s >= 1.20 
```

### CODE REVIEW

**Comment by qinqon** [comment]
/hold
Community has to decidy if bump and migrate or just migrate at least to unblock k8s-1.20 lane.

File: pkg/virtctl/expose/expose.go
Comment (maiqueb) [inline pkg/virtctl/expose/expose.go:246]: Could you export this to a function ? 

`func getServerVersion(virtClient) (*ServerVersion, error) {...}` (I haven't checked the types exist... )

File: pkg/virtctl/expose/expose.go
Comment (maiqueb) [inline pkg/virtctl/expose/expose.go:249]: Leftover: remove the commented line.

File: pkg/virtctl/expose/expose.go
Comment (maiqueb) [inline pkg/virtctl/expose/expose.go:264]: I'd also export this into a function, something like `morphServiceCreationRequest` or similar.

File: pkg/virtctl/expose/expose.go
Comment (maiqueb) [inline pkg/virtctl/expose/expose.go:276]: I would furthermore create a new function `CreateService` function that would call the function above and would create these new types of services. I am struggling with naming this new `CreateService` function - create opposite of legacy service type is really bad ...

**Review by maiqueb — CHANGES_REQUESTED** [review]
I learnt something with this PR, was not aware of this unstructured option to create the resources.

I would encapsulate the proposed code into a couple of functions.

**Pull request closed without merging**
